### DISCUSSION TITLE
USBの接続、TRRSの接続が安定しない

### DISCUSSION THREAD

**Discussion by ttizze** [Yowkees/keyball#614] [Q&A]
keyball44を組み立てました。
接続が安定しません。
PCに接続すると、たまに接続が成功するのですが、大体以下のような現象が起こります。
どの場合もPCから接続音はします。

1. 左右OLED無反応、左右キーボード無反応
2. 右OLED反応、左右キーボード無反応
3. 右OLED反応、右キーボード反応、左OLED無反応、左キーボード無反応

どこかのはんだが悪いのかと思うのですが、こういう場合どこが悪いのか教えていただけると幸いです。
よろしくお願いします。

![image](https://github.com/Yowkees/keyball/assets/29332792/bc67c312-dcff-4e01-b563-2058ff84d327)
![image](https://github.com/Yowkees/keyball/assets/29332792/0e15e104-0177-4d3d-b8ea-e952606d6d1b)

![image](https://github.com/Yowkees/keyball/assets/29332792/7161345a-9a81-4a42-9e04-227706360471)

![image](https://github.com/Yowkees/keyball/assets/29332792/128713d5-29c5-4dfd-93a8-21f1c4804b2c)

**Comment by AK-yonnana** [comment]
![keyball](https://github.com/Yowkees/keyball/assets/77046594/1b1f61d0-2d50-4ab9-b6ec-41f29f1dee48)
右手ボードの赤矢印部分のはんだ付けは不要に思います。
また左基盤のPromicro１２ピンと基盤間のはんだ付けがされていないように見えます。私ははんだ付けすることで安定したのでご参考まで。

**Reply by ttizze** [reply]
ありがとうございます。
そこははんだが流れてしまって取れなかったのでそのままにしたのですが、取ったほうが安定する可能性はあるでしょうか？

promicroと基盤は、ビルドガイドでははんだ付けしないと書かれていたのですが、どこをはんだ付けすればよいとかありますか？全部でしょうか？

ご教授いただければ幸いです。

**Reply by AK-yonnana** [reply]
確かにビルドガイドでははんだ付けしないとなっていますね。半田付けしないと確かに取り外せるので便利です。ただ各接点が接触するかどうかは保証できないですね。私もうまく接触しなくてOLEDも点灯しなかったので、はんだ付けしました。
Promicroと基板間のはんだ付けは一か所でもすると取りはずせなくなるので、やるなら全部ですね。その場合はんだ吸取り機やヒートガンなどがないと取り外せなくなるので自己責任でお願いします。
